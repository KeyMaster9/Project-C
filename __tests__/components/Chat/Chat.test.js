import React from 'react';
import renderer from 'react-test-renderer';

beforeEach(() => {
    jest.restoreAllMocks();
});

it('should render a TravelAgent Component', () => {
    jest.mock('../../../src/components/Chat/ChatItems/Messages/Info', () => ({message}) => <span>{message}</span>);
    jest.mock('../../../src/components/Chat/ChatItems/Messages/Question', () => ({message}) => <span>{message}</span>);
    jest.mock('../../../src/components/Chat/ChatItems/Messages/Response', () => ({message}) => <span>{message}</span>);

    const { default: Chat } = require('../../../src/components/Chat/Chat.jsx');
    const component = renderer.create(<Chat />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});