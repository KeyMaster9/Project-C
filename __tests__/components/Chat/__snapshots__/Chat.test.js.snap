// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a TravelAgent Component 1`] = `
<div
  className="Chat"
>
  <span>
    Info message
  </span>
  <span>
    Question!
  </span>
  <span>
    Response!
  </span>
  <div />
</div>
`;
